packages/sdks/web-component/src/lib/helpers/templates.ts

```diff
@@ -303,7 +303,9 @@ export const setPhoneAutoDetectDefaultCode = (
   autoDetectCode?: string,
 ) => {
   Array.from(
-    fragment.querySelectorAll('descope-phone-field[default-code="autoDetect"]'),
+    fragment.querySelectorAll(
```

Comment:
@nirgur We might as well remove the component name and replace this query with `[default-code="autoDetect"]` - WDYT?

Reply:
Let's do it

Reply:
Changed.